packages/editor/src/components/inserter-sidebar/style.scss

```diff
@@ -7,15 +7,26 @@
 }
 
 .block-editor-inserter-sidebar__header {
-	border-bottom: $border-width solid $gray-300;
-	padding-right: $grid-unit-10;
 	display: flex;
 	justify-content: space-between;
+	padding: $grid-unit-05 $grid-unit-05 0 $grid-unit-20;
+	background-color: $gray-100;
+}
 
-	.block-editor-inserter-sidebar__close-button {
-		order: 1;
-		align-self: center;
-	}
+.block-editor-inserter-sidebar__header-title {
+	font-size: 12px;
+	font-weight: 500;
+	color: #6b6b6b;
```

Comment:
@richtabor @SaxonF Had to change this from $gray-600 to meet AA contrast requirement of 4.5:1. Feel free to take this over and apply the background and text color styles you want.

Reply:
@jameskoster - I meant to ping you on this about the colors. I quite like your design, and we only need to tweak the colors to meet AA contrast requirements.

Reply:
A white background would make $gray-700 adequate, this might be a better approach?